website/decorators.py

```diff
@@ -0,0 +1,34 @@
+from flask import make_response
+from functools import wraps
+
+
+def add_response_headers(headers=None, default_headers=None):
```

Comment:
This is really clean, and decorators are really confusing. Nicely done! Maybe a more descriptive name; `response_security_headers` or something?